src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -440,6 +440,7 @@ public function formAction($id, Request $request)
         // Prepare combination form (fake but just to validate the form)
         $combinations = $modelMapper->getAttributesResume();
         if (is_array($combinations)) {
+            ini_set('max_input_vars', 200 + (20 * count($combinations)));
```

Comment:
Are you sure setting this during runtime works?

Reply:
yes, or it was luck because I've tried by reducing the value I had on my own php.ini file.

I'm not sure throwing an error here can help the merchant, it's like saying to him it's his fault because he have too much combinations when it's a core feature to be able to manage an undefined number of combinations per product. To be honest, 170 combinations is not a lot: I've just used the combination generator on the few attributes we provide: it's really far away from every merchant needs and some of them have 500 or 1000 combinations for a product.

Maybe we should enforce a bigger `max_input_vars` value during the installation, I don't know.